haystack/nodes/answer_generator/transformers.py

```diff
@@ -192,8 +192,9 @@ def _get_contextualized_inputs(
             truncation=True,
         )
 
-        return contextualized_inputs["input_ids"].to(self.devices[0]), contextualized_inputs["attention_mask"].to(
-            self.devices[0]
+        return (
+            contextualized_inputs["input_ids"].to(self.devices[0]),
+            contextualized_inputs["attention_mask"].to(self.devices[0]),
```

Comment:
How is this related to the output fix? 🤔

Reply:
this was introduced by black formatting. =.= I thought about this change before committing. But in order to pass the black test, I committed this change... should I revert it back?